File: lalxml/swig/octave/check-swiglalxml.m
## check SWIG Octave module wrapping the LALXML library
## Author: Karl Wette, 2011

function msg(varargin)
  disp([program_name, ": ", sprintf(varargin{:})]);
endfunction

## check that the module loads
try
  addpath(pwd(), 0);
  swiglalxml;
catch
  msg("FAILED module load");
  exit(1);
end_try_catch

## passed all tests!
msg("================");
msg("PASSED all tests");
msg("================");
exit(0)

File: lalxml/swig/python/check-swiglalxml.py
# check SWIG Python module wrapping the LALXML library
# Author: Karl Wette, 2011

import sys, os

def msg(str):
    print(os.path.basename(__file__) + ": " + str)

# check that the module loads
try:
    sys.path.insert(0, os.getcwd())
    from swiglalxml import *
except:
    msg("FAILED module load")
    exit(1)

# passed all tests!
msg("================")
msg("PASSED all tests")
msg("================")
exit(0)
